connector/connect/common/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -189,6 +190,11 @@ message Expression {
       int32 days = 2;
       int64 microseconds = 3;
     }
+
+    message Array {
+      DataType elementType = 1;
```

Comment:
Why do we need the element type? The element should be descriptive enough to figure this out.

Reply:
nvm this does not work for empty arrays.